### PR TITLE
Ignore missing stat keys

### PR DESCRIPTION
**Pull request by zacblazic**
Prevents missing `stat` keys from short-circuiting the scrape function, which perviously resulted in:
* Metrics further down not being scraped
* Reporting failed scrapes
* Reporting resque being as down

### CODE REVIEW

File: resque_exporter.go [hunk 1]
```diff
@@ -204,13 +204,13 @@ func (e *Exporter) scrape(ch chan<- prometheus.Metric) error {
 	}(time.Now())
 
 	executions, err := e.redisClient.Get(e.redisKey("stat:processed")).Float64()
-	if err != nil {
+	if err != nil && err != redis.Nil {
```
Comment (itskingori) [inline resque_exporter.go]: > Because if there is no job activity, these keys won't exist yet.

@zacblazic Could you explain this further? 🤔

Comment (zacblazic) [reply resque_exporter.go]: So these `stat` keys are only created on redis when, for example in the case of `stat:processed`, a job is completed. It's likely that this key will be created pretty soon after a redis restart or flush. However, a key like `stat:failed` will only be created when a job fails, quite a bit less likely.

Comment (itskingori) [reply resque_exporter.go]: In the case that `stat:processed` and `stat:failed` don't exist (yet) we get `redis.Nil`?

Comment (zacblazic) [reply resque_exporter.go]: Correct, see: https://godoc.org/github.com/go-redis/redis#Client.Get

Comment (itskingori) [reply resque_exporter.go]: Understood!

**Pull request merged**